src/infrastructure/managing-items.md

```diff
@@ -73,38 +73,55 @@ The necessary context object is available in all controllers and clients via `$t
 
 ## Create item
 
-Creating a new empty item is the easiest part. Simply call *create() of the manager you need a new item from:
+Creating a new empty item is the easiest part. Simply call the `create()` method of the manager you need a new item from:
 
 ```php
 $item = $manager->create();
 $item = $manager->create( [/*...*/] );
 ```
 
-This creates a new object with some default values for the item set. You can add properties if you pass a list of key/value pairs in the same structure as returned by toArray() of the item as first argument. The item doesn't have an unique ID by default and is only available in memory.
+This creates a new object with some default values for the item set. You can add properties if you pass a list of key/value pairs:
+
+```php
+$item => $manager->create( [$key1 => $value1, $key2 => $value2] )
```

Comment:
This is misleading because non of the variables is defined anywhere

Reply:
But $key and $value are generics, placeholder that simply fit the purpose to demonstrate something...

Reply:
Then use `['key1' => 'value1', 'key2' => 'value2']` but not `[$key1 => $value1, $key2 => $value2]`. People will complain that the code won't work